spec/models/person_spec.rb

```diff
@@ -1,10 +1,34 @@
 require 'rails_helper'
 
-describe Person, type: :model do
-  subject { FactoryGirl.create(:person) }
-
-  describe 'creates a people' do
-    it { is_expected.not_to be_new_record }
-    it { is_expected.to be_valid }
+describe Person do
+  it 'is valid with a name and height' do
+    person = Person.new(
```

Comment:
Awesome! You use `new` correctly here. A `create` is what most people use and would create a new database record - imagine doing thousands of those in your tests, it would become slow quickly.